tez-tools/pom.xml

```diff
@@ -50,6 +50,25 @@
         <module>tez-tfile-parser</module>
       </modules>
     </profile>
+    <profile>
+      <id>jdk1.8</id>
+      <activation>
+        <jdk>1.8</jdk>
+      </activation>
+      <modules>
+        <module>tez-javadoc-tools-base-jdk8</module>
+      </modules>
+    </profile>
+    <profile>
+      <id>jdk9plus</id>
+      <activation>
+        <!-- JDK9 starts strong encapsulation of Java internals, JDK17 removes/hides those -->
+        <jdk>[9,9999)</jdk>
```

Comment:
which version does this expression target?
if every version starting with "9", then a simply "9" should work
https://maven.apache.org/guides/introduction/introduction-to-profiles.html
```
The following configuration will trigger the profile when the JDK's version starts with "1.4" (eg. "1.4.0_08", "1.4.2_07", "1.4"):

<profiles>
  <profile>
    <activation>
      <jdk>1.4</jdk>
    </activation>
    ...
  </profile>
</profiles>
```

Reply:
This is a range of versions, based on https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html I wanted to write for JDK version 9 and up:
` <jdk>[9,)</jdk>`

It also could be 17 and up as only 17 enforces the new API.